playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -29,6 +29,10 @@
       {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }} --force --delete-local-data
     delegate_to: "{{ groups.oo_first_master.0 }}"
 
+  - name: Pause for pod rescheduling during node ugprade
+    pause:
+      seconds: "{{ openshift_node_upgrade_pause | default(30) | int }}"
+
```

Comment:
I'm trying to think if this is the right place or not. 

If we had say 3 compute nodes, and 2 replicas of some pod on N1 and N2, N3 is idle. We drain N1 for upgrade, kubernetes will kill our N1pod, and shortly thereafter realize it could distribute a replicate to N3. It might however take time to come up on N3 pulling images and such. So the problem arises if we proceed to upgrade N1 too quickly, then move on to N2 and drain it, killing our last running replica because N3 is still busy doing something to get the pod up, and now our app is down. 

Is this what we think happened? Just trying to keep it straight because this stuff can be confusing. 

Pausing after the drain seems reasonable, I'm wondering if it would be better to pause at the end of the node upgrade role instead though, as that would mean we only do this in one place, and could define a default for the seconds in the role. However it does feel like this is an "orchestration" pause and perhaps something that should be handled by the caller playbook. 

PR as is looks good to me, but throwing the above suggestion out there for comment.